### PR TITLE
Fix/Docs

### PR DESCRIPTION
**Pull request by danstarns**
Updating the readmes to include the new async schema generation.

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-1201.surge.sh

File: docs/modules/ROOT/pages/guides/migration-guide/server.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ const typeDefs = require("./type-definitions");
 
 const neo4jGraphQL = new Neo4jGraphQL({ typeDefs });
 
-const schema = neo4jGraphQL.schema;
+const schema = await neo4jGraphQL.generateSchema();
```
Comment (darrellwarde) [inline docs/modules/ROOT/pages/guides/migration-guide/server.adoc:28]:
```suggestion
const schema = await neo4jGraphQL.getSchema();
```

File: packages/graphql/README.md [hunk 2]
```diff
@@ -59,17 +59,22 @@ const typeDefs = `
...
 const driver = neo4j.driver("bolt://localhost:7687", neo4j.auth.basic("neo4j", "letmein"));
 
 const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
 
-const server = new ApolloServer({
-    schema: neoSchema.schema,
-    context: ({ req }) => ({ req }),
-});
+async function main() {
+    const schema = await neoSchema.generateSchema();
```
Comment (darrellwarde) [inline packages/graphql/README.md:67]:
```suggestion
    const schema = await neoSchema.getSchema();
```

**Review by darrellwarde — APPROVED** [review]
Nice catch, I'll just apply these changes to get the function name right!

**Review by darrellwarde — APPROVED** [review]

**Pull request closed without merging**